packages/reports/addon/components/navi-warning.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{navi-warning
```

Comment:
Can this be reused for any navi warning msg? Otherwise it might be a good idea to have a name more aligned with its function.

Reply:
That was the idea. We use it right now for missing intervals, but you could override the warning message to display something different.